spec/item_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './item'
+
+describe Item do
```

Comment:
- [ ] Good job adding tests to the `Item` class 👏.  However, I think there are some methods that are not tested yet: the `move_to_archive`,  the `genre`, the `author`  and the `label` methods. As per the requirement and as reminded by the previous reviewer, all methods inside the classes should be tested. So I would kindly suggest you test those four methods too.  🤗 👌

### Requirement 
![Screenshot from 2023-11-03 14-59-27](https://github.com/MayPyone/Catalog-of-my-things/assets/89228743/3f37f473-3362-449e-8c74-312d09deb5d9)